src/extractor/ValueSetConceptComponentRuleExtractor.ts

```diff
@@ -0,0 +1,39 @@
+import { fshtypes } from 'fsh-sushi';
+import { cloneDeep } from 'lodash';
+import { ExportableValueSetConceptComponentRule } from '../exportable';
+import { ProcessableValueSetComponent } from '../processor';
+
+export class ValueSetConceptComponentRuleExtractor {
+  static process(
+    vsComponent: ProcessableValueSetComponent,
+    include: boolean
+  ): ExportableValueSetConceptComponentRule {
+    if (vsComponent.concept?.length > 0) {
+      const rule = new ExportableValueSetConceptComponentRule(include);
+      rule.from.system = vsComponent.system;
+      rule.from.valueSets = cloneDeep(vsComponent.valueSet);
+      vsComponent.concept.forEach(concept => {
+        if (ValueSetConceptComponentRuleExtractor.isConceptComponent(concept)) {
```

Comment:
Technically it's not valid FHIR if `code` isn't there.  I wouldn't be opposed to just skipping this check, because I actually think it's ok (and maybe even better) if the result reflects the fact that there was invalid FHIR source.  As it is now, we'd just swallow that.

Reply:
Agreed. I'll go put another quarter in the "goFSH is not a FHIR validator" jar, because I seem to have forgotten that important fact yet again.